docs/guide/advanced/navigation-guards.md

```diff
@@ -167,6 +167,7 @@ beforeRouteLeave (to, from, next) {
 7. Call `beforeRouteEnter` in activated components.
 8. Call global `beforeResolve` guards.
 9. Navigation confirmed.
+1. Call global `onReady` hooks (Only during initial navigation).
```

Comment:
Exact numbering is not required in markdown. Normally to make a numbered list I would just make everything `1. ` in order to avoid the delta showing all subsequent items in the list as changed. If the team is okay with that, I'll update the numbering accordingly.